pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -165,6 +165,8 @@ func (r *OnlineUpgradeReconciler) Reconcile(ctx context.Context, _ *ctrl.Request
 
 	// Functions to perform when the image changes.  Order matters.
 	funcs := []func(context.Context) (ctrl.Result, error){
+		// Requeue if not all nodes are running
+		r.requeuePodsNotRunning,
```

Comment:
Add a status msg in `onlineUpgradeStatusMsgs` for this step.

Reply:
Done.

Reply:
Let's put them after postStartOnlineUpgradeMsg